pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -578,28 +580,30 @@ self: super: {
     dependencies = with self; [ plenary-nvim ];
   });
 
-  markdown-preview-nvim = super.markdown-preview-nvim.overrideAttrs (old: let
-    # We only need its dependencies `node-modules`.
-    nodeDep = nodePackages."markdown-preview-nvim-../../applications/editors/vim/plugins/markdown-preview-nvim".overrideAttrs (old: {
-      dontNpmInstall = true;
-    });
-  in {
-    patches = [
-      (substituteAll {
-        src = ./markdown-preview-nvim/fix-node-paths.patch;
-        node = "${nodejs}/bin/node";
-      })
-    ];
-    postInstall = ''
-      ln -s ${nodeDep}/lib/node_modules/markdown-preview/node_modules $out/app
-    '';
+  markdown-preview-nvim = super.markdown-preview-nvim.overrideAttrs (old:
```

Comment:
Starting from this line, changes come from `nixpkgs-fmt`.

Reply:
:| They'll likely ask you not to run `nixpkgs-fmt` on the file even though a _lot_ of the code *does* use that formatter and/or style (though I'm sure there's a lot of us that would prefer the whole repo used a single formatter for consistency). You could reset the commit and try `git add -p` and just patch in your lines.

Reply:
Changes reverted!